src/dailies.ts

```diff
@@ -145,7 +149,53 @@ export function postFreeFightDailySetup(): void {
 }
 
 function voterSetup(): void {
-  if (have($item`"I Voted!" sticker`) || !(get("voteAlways") || get("_voteToday"))) return;
+  if (have($item`"I Voted!" sticker`)) return;
+
+  const initPriority: Map<string, number> = new Map([
+    [
+      "Meat Drop: +30",
+      0.3 *
+        ((baseMeat + 750) * embezzlerCount() + baseMeat * (estimatedTurns() - embezzlerCount())),
+    ],
+    [
+      "Item Drop: +15",
+      0.15 *
+        (4 * 100 * 0.3 * embezzlerCount() + 3 * 200 * 0.15 * (estimatedTurns() - embezzlerCount())),
+    ],
+    ["Adventures: +1", globalOptions.ascending ? 0 : get("valueOfAdventure")],
+    ["Familiar Experience: +2", 8],
+    ["Monster Level: +10", 5],
+    [`${myPrimestat()} Percent: +25`, 3],
+    [`Experience (${myPrimestat()}): +4`, 2],
+    ["Meat Drop: -30", -2],
+    ["Item Drop: -15", -2],
+    ["Familiar Experience: -2", -2],
+  ]);
+
+  if (!get("voteAlways") && !get("_voteToday")) {
+    const availableInitiatives: Map<string, number> = new Map(
+      Object.keys(votingBoothInitiatives(myClass(), myPathId(), myDaycount())).map((init) => {
+        const val = initPriority.get(init) ?? 0;
+        return [init, val];
+      })
+    );
+
+    const ballotValue =
+      sum(
+        Object.values(availableInitiatives)
+          .sort((a, b) => b - a)
+          .slice(1),
+        (val) => 2 * val
+      ) +
+      3 * get("garbo_valueOfFreeFight", 2000);
+
+    if (
+      ballotValue > mallPrice($item`absentee voter ballot`) &&
+      acquire(1, $item`absentee voter ballot`, ballotValue)
```

Comment:
```suggestion
      acquire(1, $item`absentee voter ballot`, ballotValue, false)
```
we really need to update the behavior of `acquire`